components/BottomSheet/src/private/MDCSheetContainerView.m

```diff
@@ -239,6 +239,10 @@ - (void)updateSheetFrame {
     // If the content doesn't scroll then we have to set its frame to the size we are making
     // visible. This ensures content using autolayout lays out correctly.
     contentFrame.size.height = [self truncatedPreferredSheetHeight];
+  } else {
+    // If the content does scroll, don't make it larger than the content height or preferred height.
+    contentFrame.size.height =
+        MAX(self.sheet.scrollView.contentSize.height, [self truncatedPreferredSheetHeight]);
```

Comment:
Should this be MIN, based on the comment?

Reply:
I know it might be a pain but this change is potentially risky so it may be worth pulling out to a separate PR as well.